labs-ios-starter/Login/LoginViewController.swift

```diff
@@ -57,11 +57,30 @@ class LoginViewController: UIViewController {
             guard let self = self,
                 self.presentedViewController == nil else { return }
             
+            // This code will find the story board and locate the inital tab bar screen and present that if the user's login was sucessfull.
+            
+            let storyBoard = UIStoryboard(name: "Main", bundle: nil)
+            
+            let tabBarViewController = storyBoard.instantiateViewController(withIdentifier: "InitalTabBarView")
+
             if exists {
-                self.performSegue(withIdentifier: "ShowDetailProfileList", sender: nil)
+                self.navigationController?.pushViewController(tabBarViewController, animated: true)
```

Comment:
This push is causing there to be a back button to be shown that we don't want (right? the user doesn't need to get back to that screen, unless they log out, and we will have a logout button on the profile screen). 

Can you try doing a modal present instead? Could be animated, or not.

<img width="469" alt="Screen Shot 2021-01-27 at 9 08 50 PM" src="https://user-images.githubusercontent.com/438307/106093388-0a35b000-60e5-11eb-8684-55e5a0d2f0c5.png">